actions/st2_perform_migrations.sh

```diff
@@ -153,6 +153,13 @@ migrate_to_1.5() {
   run_migration_scripts $scripts
 }
 
+migrate_to_2.2() {
```

Comment:
Change this to update_mistral_db and add a line in line 167 to perform database upgrade.